packages/expo-contacts/src/Contacts.js

```diff
@@ -342,78 +341,76 @@ export function addExistingGroupToContainerAsync(
   groupId: string,
   containerId: string
 ): Promise<any> {
-  if (isIos) {
+  if (ExpoContacts.addExistingGroupToContainerAsync) {
```

Comment:
Style: handle the early-return edge cases first so we can prune down the state space of the program and have less indentation for the real "meat" of the function ex:

```js
if (!ExpoContacts.addExistingGroupToContainerAsync) {
  throw new Error(...);
}

// main implementation here
```

This is one of the things in our guide to writing modules, I'll send you a link separately.

Reply:
Actually for calls like `if (method) { method() }` where it's just one or two lines that are guarded by the check, I think the way you have it here is better.

(And for larger blocks of code, the early return keeps the function less nested.)